lambdas/service/app.py

```diff
@@ -85,11 +85,71 @@
         {
             'name': 'Health',
             'description': 'For checking on service health'
+        },
+        {
+            'name': 'Version',
+            'description': 'Describes the current version of Azul service'
         }
     ],
     'servers': [
         {'url': config.service_endpoint()}
-    ]
+    ],
+    'components': {
+        'responses': {
+            'OkVersion': {
```

Comment:
Why is this extracted into a component? AFAICT, it doesn't need to be since there is only one reference. 

Related things should be in close proximity unless the value of reusing something or abstracting it outweighs the proximity concern. There is no abstraction or reuse in this case it seems.